rust/gui-client/src-tauri/src/client/logging.rs

```diff
@@ -175,7 +175,7 @@ fn choose_logs_to_delete(paths: &[PathBuf]) -> Vec<&Path> {
                 return false;
```

Comment:
If you'd use a `filter_map` right away you could use `?` for these two which is a bit more concise. And for the boolean you could then use `.then_some` or `.then`